README.md

```diff
@@ -6,6 +6,28 @@ An Ansible role for installing [Terraform](https://terraform.io/).
 
 - `terraform_version` - Terraform version
 
-## Example Playbook
+## Testing
+Tests are done using [molecule](http://molecule.readthedocs.io/). To run the test suite, install molecule and its dependencies and run ` molecule test` from the folder containing molecule.yml. To add additional tests, add a [testinfra](http://testinfra.readthedocs.org/) python script in the [tests](./tests/) directory, or add a function to [test_terraform.py](./tests/test_terraform.py). Information about available Testinfra modules is available [here](http://testinfra.readthedocs.io/en/latest/modules.html).
 
-See the [examples](./examples/) directory.
+### Example 
+```
+# Download molecule, dependencies
+$ pip install molecule
+
+# Change to the top-level project directory, which contains molecule.yml
+$ cd /path/to/ansible-terraform
+
+# Ensure that molecule.yml is present
+$ ls
+CHANGELOG.md                             molecule.yml
+LICENSE                                  playbook.retry
+README.md                                playbook.yml
+ansible.cfg                              tasks
+defaults                                 templates
+handlers                                 tests
+meta                                     
+
+# We're in the right directory, so let's run tests!
+$ molecule converge && molecule test
```

Comment:
I think maybe this can be just `molecule test` again. Before it didn't seem to be installing the dependencies before the test run, but now it appears to be doing so. Does just `molecule test` work end-to-end for you?